ocs_ci/deployment/aws.py

```diff
@@ -402,6 +422,386 @@ def deploy_ocp(self, log_cli_level='DEBUG'):
+                host, pem_dst_path, disable, user=self.rhel_worker_user
+            )
+            rhel_pod_obj.copy_to_server(
+                host, pem_dst_path, f'{repo_dst_path}/{repo_file}',
+                f'/tmp/{repo_file}', user=self.rhel_worker_user
+            )
+            rhel_pod_obj.exec_cmd_on_node(
+                host, pem_dst_path,
+                f'sudo mv /tmp/{repo_file} {repo_dst_path}',
+                user=self.rhel_worker_user
+            )
+            rhel_pod_obj.copy_to_server(
+                host, pem_dst_path,
+                f'{dst}/{constants.INTERNAL_MIRROR_PEM_FILE}',
+                f'/tmp/{constants.INTERNAL_MIRROR_PEM_FILE}',
+                user=self.rhel_worker_user,
+            )
+            rhel_pod_obj.exec_cmd_on_node(
+                host, pem_dst_path,
+                f'sudo mv /tmp/{constants.INTERNAL_MIRROR_PEM_FILE} {dst}',
+                user=self.rhel_worker_user
+            )
+        # copy kubeconfig to pod
+        kubeconfig = os.path.join(
+            self.cluster_path, config.RUN.get('kubeconfig_location')
+        )
+        pod.upload(rhel_pod_obj.name, kubeconfig, "/")
+        pull_secret_path = os.path.join(
+            constants.TOP_DIR,
+            "data",
+            "pull-secret"
+        )
+        pod.upload(rhel_pod_obj.name, pull_secret_path, "/tmp/")
+        host_file = self.build_ansible_inventory(hosts)
+        pod.upload(rhel_pod_obj.name, host_file, "/")
+        # install pod packages
+        rhel_pod_obj.install_packages(constants.RHEL_POD_PACKAGES)
+        # run ansible
+        openshift_ansible_path = "/usr/share/ansible/openshift-ansible"
+        cmd = (
+            f"ansible-playbook -i /hosts --private-key={pem_dst_path} "
+            f"{openshift_ansible_path}/playbooks/scaleup.yml"
+        )
+
+        rhel_pod_obj.exec_cmd_on_pod(
+            cmd, out_yaml_format=False, timeout=timeout
+        )
+        self.verify_nodes_added(hosts)
+        # remove rhcos workers
+        self.remove_rhcos_workers()
+
+    def remove_rhcos_workers(self):
+        """
+        After RHEL workers are added remove rhcos workers from the cluster
+
+        Raise:
+            FailedToRemoveNodeException: if rhcos removal is failed
+        """
+        rhcos_workers = self.get_rhcos_workers()
+        for node in rhcos_workers:
+            cordon = f"oc adm cordon {node}"
+            run_cmd(cordon)
+            drain = (
+                f"oc adm drain {node} --force --delete-local-data "
+                f"--ignore-daemonsets"
+            )
+            run_cmd(drain)
+            delete = f"oc delete nodes {node}"
+            run_cmd(delete)
+        if len(self.get_rhcos_workers()):
+            raise exceptions.FailedToRemoveNodeException()
+
+    def get_rhcos_workers(self):
+        """
+        Returns a list of rhcos worker names
+
+        Returns:
+            rhcos_workers (list): list of rhcos worker nodes
```

Comment:
Leave an empty line at the end of the docstring.

Reply:
Done